swap_meet/electronics.py

```diff
@@ -1,2 +1,12 @@
-class Electronics:
-    pass
+
+from .item import Item
+
+class Electronics(Item):
+    def __init__(self, id = None, condition = None, type = None):
+        super().__init__(id, condition)
+        self.type = "Unknown" if type == None else type
+    
+    def __str__(self):
+        return f"An object of type Electronics with id {self.id}. This is a {self.type} device."
```

Comment:
Check out my comment in clothing.py about how to use the parent class method in this method.